index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>favour's porfolio page</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header id="toolbar">
+      <a href="index.html"
+        ><img
+          id="logoimage"
+          src="images/image_geometry_1.png"
+          alt="My portfolio's logo"
+      /></a>
+      <nav id="menuimage">
+        <img src="./images/ic_menu.svg" alt="menu icon" />
+      </nav>
+    </header>
+    <section class="headline">
+      <h1 class="name">Favour <br />Ezeugwa</h1>
+      <p class="bio">
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Taka a look of my works. If you like what you see
+        and have a project you need coded, don't hesitate and contact me.
+      </p>
+      <button type="button" class="profilelin">
+        <span class="toprofile">Start collaboration</span>
+      </button>
+    </section>
+    <div>
+      <img
+        id="Geometry2"
+        src="./images/image_geometry_2.png"
+        alt="porfolio theme"
+      />
```

Comment:
- [ ] [OPTIONAL] try to use this image as a background image not as a `<img>` tag because it's only for design purposes